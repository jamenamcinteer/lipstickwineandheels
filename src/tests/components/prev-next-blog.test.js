import React from 'react'
import { shallow } from 'enzyme'
import PrevNextBlog from '../../components/prev-next-blog'
import data from '../fixtures/data'

test('should render article preview component correctly', () => {
  const wrapper = shallow(
    <PrevNextBlog
      prevPost={data.data.allContentfulBlog.edges[0]}
      nextPost={data.data.allContentfulBlog.edges[1]}
    />
  )
  expect(wrapper).toMatchSnapshot()
})
