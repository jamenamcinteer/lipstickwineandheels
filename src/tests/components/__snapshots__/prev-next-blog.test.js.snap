// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render article preview component correctly 1`] = `
<ul
  className="prevNext__container"
>
  <li
    className="prevNext__itemPrev"
  >
    <GatsbyLink
      className="button button-secondary prevNext__link"
      to="/blog/this-is-a-sample-blog-post"
    >
      <div
        className="prevNext__contentContainer"
      >
        <span
          className="prevNext__heading"
        >
          Previous
        </span>
        <h2
          className="prevNext__title"
        >
          This is a sample blog post
        </h2>
        <p
          className="prevNext__date"
        >
          August 30, 2018
        </p>
      </div>
      <div
        className="prevNext__imageContainer"
      />
    </GatsbyLink>
  </li>
  <li
    className="prevNext__itemNext"
  >
    <GatsbyLink
      className="button button-secondary prevNext__link"
      to="/blog/hello-world"
    >
      <div
        className="prevNext__imageContainer"
      />
      <div
        className="prevNext__contentContainer"
      >
        <span
          className="prevNext__heading"
        >
          Next
        </span>
        <h2
          className="prevNext__title"
        >
          Hello World!
        </h2>
        <p
          className="prevNext__date"
        >
          August 28, 2018
        </p>
      </div>
    </GatsbyLink>
  </li>
</ul>
`;
